migrate/20240503_boot_image.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:boot_image) do
```

Comment:
Should we add a `created_at` column to this table? It could be useful to know when we downloaded the image to the host.
```
      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
```